.gitattributes

```
@@ -1,2 +1,4 @@
 *.rs diff=rust
 *.md diff=markdown
+*.env text eol=lf
+*.env.golden text eol=lf
```

Add newline at end of the file, please 🙂 
```suggestion
*.env.golden text eol=lf

```

---

Good catch. I'm not sure why my editor didn't add that based on the `.editorconfig` file.